test-integration/filter-weth.js

```diff
@@ -0,0 +1,60 @@
+/* global artifacts */
+
+const { expect } = require("chai");
+const utils = require("../utils/utilities.js");
+const ArgentContext = require("../utils/argent-context.js");
+
+const WethFilter = artifacts.require("WethFilter");
+const WETH = artifacts.require("WETH9");
+
+contract("WETH Filter", (accounts) => {
+  let argent, wallet;
+  let weth;
+
+  before(async () => {
+    argent = await new ArgentContext(accounts).initialize();
+
+    weth = await WETH.at("0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2");
```

Comment:
we could use `argent.WETH.address`, or use `argent.WETH` if we change use the `WETH` abi  instead of the `ERC20` abi in `argent-context.js`

Reply:
done, used the WETH abi